examples/cdn-policy/main.tf

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_network" "default" {
+  name                    = var.network_name
+  project                 = var.project_id
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name                     = var.network_name
+  project                  = var.project_id
+  ip_cidr_range            = "10.127.0.0/20"
+  network                  = google_compute_network.default.self_link
+  region                   = var.region
+  private_ip_google_access = true
+}
+
+resource "google_compute_router" "default" {
+  name    = "lb-http-router"
+  project = var.project_id
+  network = google_compute_network.default.self_link
+  region  = var.region
+}
+
+module "cloud-nat" {
+  source     = "terraform-google-modules/cloud-nat/google"
+  version    = "~> 2.2"
+  router     = google_compute_router.default.name
+  project_id = var.project_id
+  region     = var.region
+  name       = "cloud-nat-lb-http-router"
+}
+
+data "template_file" "group-startup-script" {
```

Comment:
nit: https://developer.hashicorp.com/terraform/language/functions/templatefile is now preferred